app/views/keywords/_scrape_result_search_form.erb

```diff
@@ -0,0 +1,24 @@
+<%= form_tag(
+  search_keywords_path,
+  method: 'get',
+  class: 'scrape-result-search-form',
+  data: {"testid" => "scrape-result-search-form"}
+) do %>
+  <%= select_tag(
+    :query_type,
+    options_for_select([
+      [I18n.t('keywords.query_type_select_placeholder'), nil],
+      [I18n.t('keywords.query_types.exact'), :exact],
+      [I18n.t('keywords.query_types.partial'), :partial],
+      [I18n.t('keywords.query_types.pattern'), :pattern]
+    ]),
+    class: 'p-2'
+  ) %>
+  <%= text_field_tag(
+    :search,
+    '',
+    class: 'scrape-result-search-form__input',
+    placeholder: I18n.t('keywords.search_input_placeholder')
```

Comment:
We can add the `required: true` tothis `text_field` so the form won't be submit if the field is empty @Goose97

Reply:
Good catch. Fixed in 1cf4e0b6a57c486049f3ec6baae7f1fbb404cb7b